components/tenant-mgt/org.wso2.carbon.tenant.mgt.ui/src/main/resources/web/tenant-mgt/js/tenant_config.js

```diff
@@ -163,7 +163,7 @@ function validateDomain(fld, isPublicCloudSetup) {
         error = "Invalid domain, starting with '.'";
     }
     else if (illegalChars.test(fld.value)) {
-        error = "The domain only allows letters, numbers, '.', '-' and '_'. <br />";
+        error = "The domain only allows lower case letters and numbers only";

```

Comment:
Why is the message changed? We have to say that '.', '-' and '_' are also valid character. Basically use the same message as before, only change is instead of letters we have to say lower case letters.